docs/core/tools/dotnet-new-sdk-templates.md

```diff
@@ -25,9 +25,9 @@ The following table shows the templates that come preinstalled with the .NET SDK
 
 | Templates                   | Short name                        | Language | Tags                    | Introduced |
 |-----------------------------|-----------------------------------|----------|-------------------------|------------|
-| ASP.NET Core API            | [`webapiaot`](#webapiaot)         | [C#]     | Web/Web API/API/Service | 8.0        |
-| ASP.NET Core API controller | [`apicontroller`](#apicontroller) | [C#]     | Web/ASP.NET             | 8.0        |
-| ASP.NET Core Empty          | [`web`](#web)                     | [C#], F# | Web/Empty               | 1.0        |
+| ASP.NET Core API            | [`webapiaot`](#webapiaot)         | [C#]     | Web/Web API/API/Service | 8.0       |
+| ASP.NET Core API controller | [`apicontroller`](#apicontroller) | [C#]     | Web/ASP.NET             | 8.0       |
+| ASP.NET Core Empty          | [`web`](#web)                     | [C#], F# | Web/Empty               | 8.0        |
```

Comment:
Did you mean to change the introduced version here?

Reply:
Nope :)